source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx

```diff
@@ -133,61 +158,64 @@ const useLocationFeatureLoader = () => {
         municipalityFeatureTask,
       ]);
 
-      let pimsLocationProperties:
-        | FeatureCollection<Geometry, PIMS_Property_Location_View>
-        | undefined = undefined;
+      let pimsLocationProperties: Feature<Geometry, PIMS_Property_Location_View>[] | undefined =
+        undefined;
 
       // Load PimsProperties
       // - first attempt to find it by our internal PIMS id
       // - then try to find it on our boundary layer.
       // - if not found by boundary attempt to match it by PID / PIN coming from parcel-map
       const isInPims = isValidId(Number(pimsPropertyId));
       if (isInPims) {
-        pimsLocationProperties = await loadProperties({ PROPERTY_ID: Number(pimsPropertyId) });
+        pimsLocationProperties = (await loadProperties({ PROPERTY_ID: Number(pimsPropertyId) }))
+          .features;
       } else {
-        const boundaryFeature = await findOneByBoundary(latLng, 'GEOMETRY', 4326);
-        if (exists(boundaryFeature)) {
-          pimsLocationProperties = { features: [boundaryFeature], type: 'FeatureCollection' };
-        } else if (
-          exists(parcelFeature) &&
-          (exists(parcelFeature.properties?.PID) || exists(parcelFeature.properties?.PIN))
-        ) {
-          pimsLocationProperties = await loadProperties({
-            PID: parcelFeature.properties?.PID || '',
-            PIN: parcelFeature.properties?.PIN?.toString() || '',
-          });
+        const boundaryPimsFeature = await findAllByBoundary(latLng, 'GEOMETRY', 4326);
+        if (exists(boundaryPimsFeature)) {
+          pimsLocationProperties = boundaryPimsFeature;
+        } else if (exists(parcelFeatures)) {
+          pimsLocationProperties = [];
+
+          for (let i = 0; i < parcelFeatures.length; i++) {
+            const parcelFeature = parcelFeatures[i];
+            if (exists(parcelFeature.properties?.PID) || exists(parcelFeature.properties?.PIN)) {
+              const pimsFeatures = await loadProperties({
```

Comment:
putting this await in this for loop will run all of these loadProperties requests serially.